README.md

```diff
@@ -23,15 +23,11 @@ This library can be installed using the normal npm workflow:
 npm install njs-memory-profiler
 ```
 
-This module will install to a folder called `njs_modules` in the root of your project. If you want to use a different directory, run the install with the `NJS_MODULES_DIR` environment variable specified:
+This module will install to a folder called `njs_modules` in the root of your project.
 
-```bash
-NJS_MODULES_DIR=./ npm install njs-memory-profiler
-```
-
-Note: As the installation will be performed by a post-install JavaScript using `node` the minimal required version of Node will be 16 or greater.
+Note: As the installation will be performed by an npm postinstall script. NodeJS version 16 or greater is required.
 
-We recommend to use [nvm](https://github.com/nvm-sh/nvm) or [asdf](https://asdf-vm.com/) to manage your node versions.
+We recommend using [asdf](https://asdf-vm.com/) or [nvm](https://github.com/nvm-sh/nvm) to manage your node versions.
```

Comment:
**annotation**: made this advice more brief, and put `asdf` before `nvm` since it's a better runtime manager for most people.